src/transaction/TransactionController.ts

```diff
@@ -316,6 +341,7 @@ export class TransactionController extends BaseController<
         from: txMeta.from,
         gas: BNToHex(new BN(txMeta.gas)),
         gasPrice: BNToHex(new BN(txMeta.gasPrice)),
+        gasUsed: BNToHex(new BN(txMeta.gasUsed)),
```

Comment:
Does gasUsed need to be added to `interface EtherscanTransactionMeta`?

Reply:
Yes, we are using this value to calculate the gas fee in the transaction history

Reply:
Sorry I'm confused, this does need to get added to the interface definition right?

Reply:
Yes, I believe this is the interface for the normalized transaction

```
export interface Transaction {
  chainId?: number;
  data?: string;
  from: string;
  gas?: string;
  gasPrice?: string;
  gasUsed?: string;
  nonce?: string;
  to?: string;
  value?: string;
  maxFeePerGas?: string;
  maxPriorityFeePerGas?: string;
  estimatedBaseFee?: string;
}
```

Reply:
Ok but here we're pulling gasUsed off of txMeta which is typed on line 313 as `EtherscanTransactionMeta` not `Transaction` so do we not have to have to add gasUsed to **that** interface?

Reply:
oh wait sorry I'm being an idiot. It's already there! 🤦